opentelekomcloud/acceptance/bms/resource_opentelekomcloud_compute_bms_tags_v2_test.go

```diff
@@ -112,9 +111,12 @@ resource "opentelekomcloud_compute_instance_v2" "instance_1" {
   image_id          = data.opentelekomcloud_images_image_v2.latest_image.id
   security_groups   = ["default"]
   availability_zone = "%s"
-  flavor_id         = "physical.o2.medium"
-  flavor_name       = "physical.o2.medium"
-  tags              = ["foo", "bar"]
+  flavor_id         = "physical.h2.large"
+  flavor_name       = "physical.h2.medium"
+  tags = {
+    foo  = "bar"
+    john = "doe"
+  }
```

Comment:
ok, then why here not tags = ["foo", "bar"]?

Reply:
https://github.com/opentelekomcloud/terraform-provider-opentelekomcloud/blob/d5b7cd5487c66237234041e1c7ad38eb3229ac77/opentelekomcloud/services/bms/resource_opentelekomcloud_compute_bms_server_v2.go#L192-L193